src/check/parse/tokenize.zig

```diff
@@ -321,14 +329,14 @@ pub const Cursor = struct {
     }
 
     /// Returns the byte at the given lookahead offset.
-    pub fn peekAt(self: *Cursor, lookahead: usize) ?u8 {
+    pub fn peekAt(self: *Cursor, lookahead: TokenIndex) ?u8 {
         if (self.pos + lookahead < self.buf.len) {
             return self.buf[self.pos + lookahead];
         }
         return null;
     }
 
-    pub fn isPeekedCharInRange(self: *Cursor, lookahead: usize, start: u8, end: u8) bool {
+    pub fn isPeekedCharInRange(self: *Cursor, lookahead: TokenIndex, start: u8, end: u8) bool {
```

Comment:
Technically lookahead isn't really an index per se, it's an offset from the current index. (that said, this will work fine - it just seems strange to me 🤷)

Reply:
Well, it's limited to 32 bits, not 64 (on some systems).  Our position is a u32

Reply:
Yeah, good point on keeping this consistent in size with the position. But I guess I'd have a slight preference for a straight u32 here, over re-using TokenIndex.